src/CloudWatchLogs/CloudWatchLogsClient.php

```diff
@@ -155,4 +156,46 @@
  * @method \Aws\Result updateLogAnomalyDetector(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateLogAnomalyDetectorAsync(array $args = [])
  */
-class CloudWatchLogsClient extends AwsClient {}
+class CloudWatchLogsClient extends AwsClient {
+    static $streamingCommands = [
+        'StartLiveTail' => true
+    ];
+
+    public function __construct(array $args)
+    {
+        parent::__construct($args);
+        $this->appendClientCustomMiddlewares();
+    }
+
+    private function appendClientCustomMiddlewares()
```

Comment:
I don't have any issues with using a separate function for this, but we might want to call this `addStreamingFlagMiddleware` and make each `add` function for middleware separate.  We won't always be appending middleware- it can be prepending or inserted before or after another middleware